src/components/review/ReviewConfiguration.jsx

```diff
@@ -58,8 +62,86 @@ const ReviewDescriptionList = ({ children }) => {
     );
 };
 
-export const ReviewConfiguration = ({ deviceData, diskSelection, language, localizationData, requests, idPrefix, setIsFormValid, storageScenarioId, accounts }) => {
+const ChangeHostname = ({ initHostname }) => {
+    const [currentHostname, setCurrentHostname] = useState("");
+    const [error, setError] = useState([]);
+    const [isModalOpen, setIsModalOpen] = React.useState(false);
+
+    const validateHostname = (value) => {
+        const error = [];
+        if (value.length > 64) { error.push(_("Real host name must be 64 characters or less")) }
+        if (value.match(/[.a-z0-9-]*/)[0] !== value || value.indexOf("..") !== -1) { error.push(_("Real host name can only contain lower-case characters, digits, dashes, and periods (with populated subdomains)")) }
```

Comment:
The match can return also null, if not matches are found. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#return_value

In this case this would crash.